src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,17 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
-        return 0;
+        String[] valuesEmailAndScore;
+        for (String record : records) {
+            valuesEmailAndScore = record.split(":");
```

Comment:
```suggestion
           String[] valuesEmailAndScore = record.split(":");
```
no need to declare it outside the loop

Reply:
Are we not saving memory?